src/triggers.js

```diff
@@ -866,13 +875,16 @@ export function maybeRunTrigger(
         resolve(object);
       },
       error => {
-        logTriggerErrorBeforeHook(
-          triggerType,
-          parseObject.className,
-          parseObject.toJSON(),
-          auth,
-          error
-        );
+        var parseObjectJSON;
+        try {
+          parseObjectJSON = parseObject.toJSON();
+        } catch (error2) {
+          // https://github.com/parse-community/parse-server/issues/7192
```

Comment:
Please remove comment; I think also `error2` is unused, so you could remove it as well?